graphql-spring-boot-test/src/main/java/com/graphql/spring/boot/test/GraphQLTestSubscription.java

```diff
@@ -315,13 +404,13 @@ public GraphQLResponse awaitAndGetNextResponse(final int timeout, final boolean
         assertThat(responses)
             .as(
                 String.format(
-                    "Expected no responses in %s MS, but received %s", timeout, responses.size()))
+                    "Expected no responses in %s, but received %s", timeout, responses.size()))
             .isEmpty();
       }
       if (numExpectedResponses > 0) {
         assertThat(responses)
             .as(
-                "Expected at least %s message(s) in %d MS, but %d received.",
+                "Expected at least %s message(s) in %d, but %d received.",
```

Comment:
Here the `%d` for `timeout` should be changed to ` %s`. And the `%s` for `numExpectedResponses` to `%d`.

Reply:
Thank you. It's dealt with